api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -252,4 +252,30 @@ public void getMostRecentQueue_shouldReturnMostRecentPatientQueue() throws Parse
 		Assert.assertEquals(patientQueue, patientQueueingService.getMostRecentQueue(patient));
 		
 	}
+	
+	@Test
+	public void generateVisitNumber_shouldGenerateVisitNumberIrrespectiveOfPreviousVisitNumberLength() {
```

Comment:
What is wrong with the previous visit number length?

Reply:
It is set Intentionally to less than what is expected.

Reply:
Why would generateVisitNumber throw an exception. I thought its role is simply generate a visit number.